Sources/TunnelKitWireGuardAppExtension/WireGuardTunnelProvider.swift

```diff
@@ -122,6 +130,28 @@ open class WireGuardTunnelProvider: NEPacketTunnelProvider {
             completionHandler(nil)
         }
     }
+
+    // MARK: Data counter (tunnel queue)
+
+    private func refreshDataCount() {
+        guard dataCountInterval > 0 else { return }
+        tunnelQueue.schedule(after: DispatchTimeInterval.milliseconds(dataCountInterval)) { [weak self] in
+            self?.refreshDataCount()
+        }
+        
+        guard tunnelIsStarted else {
+            cfg._appexSetDataCount(nil)
+            return
+        }
+        fetchDataCount { [weak self] result in
+            switch result {
+            case .success(let dataCount):
+                self?.cfg._appexSetDataCount(dataCount)
```

Comment:
Just commenting that I'm okay with a single `self?` here, just not with plenty. So again, this is okay.

Reply:
done